config.py

```diff
@@ -472,6 +472,9 @@ def create_transaction(db):
     # The version of the API to use for the security scanner.
     SECURITY_SCANNER_API_VERSION = "v1"
 
+    # Namespace whitelist for security scanner.
+    SECURITY_SCANNER_INDEXING_WHITELIST = []
```

Comment:
Name this for V4, as it'll only be used for that until we're ready for a full rollout.